packages/directlinespeech/src/createWebSpeechPonyfillFactory.js

```diff
@@ -0,0 +1,90 @@
+/* eslint class-methods-use-this: ["error", { "exceptMethods": ["cancel", "getVoices", "speak"] }] */
+
+import { AudioConfig } from 'microsoft-cognitiveservices-speech-sdk';
+import { createSpeechRecognitionPonyfillFromRecognizer } from 'web-speech-cognitive-services/lib/SpeechServices/SpeechToText';
+import AbortController from 'abort-controller';
+
+import createErrorEvent from './createErrorEvent';
+import createTaskQueue from './createTaskQueue';
+import EventTargetShim, { defineEventAttribute } from './external/event-target-shim';
+import playCognitiveServicesStream from './playCognitiveServicesStream';
+import playWhiteNoise from './playWhiteNoise';
+import SpeechSynthesisAudioStreamUtterance from './SpeechSynthesisAudioStreamUtterance';
+
+export default function({
+  audioConfig = AudioConfig.fromDefaultMicrophoneInput(),
+  enableTelemetry,
+  recognizer,
+  textNormalization
+}) {
+  return () => {
+    const { SpeechGrammarList, SpeechRecognition } = createSpeechRecognitionPonyfillFromRecognizer({
+      audioConfig,
+      createRecognizer: () => recognizer,
```

Comment:
this interface is odd to me. why do you need a `createRecognizer` function if it returns whatever was passed-in as a parameter to the outer function?